### ISSUE TITLE
Test fails on my system for DBD-Pg-3.15.1-1

### ISSUE THREAD

**Issue by cavac** [bucardo/dbdpg#94]
```
Failed test:
t/03dbmethod.t ......... 1/646 
#   Failed test 'DB handle method "get_info" returns expected result for SQL_DEFAULT_TXN_ISOLATION'
#   at t/03dbmethod.t line 624.
#          got: '8'
#     expected: '2'

```
Is that the transaction isolation level?

My /etc/postgresql/14/main/postgresql.conf on my systems have this:
default_transaction_isolation = 'serializable'

**Comment by turnstep** [comment]
Thanks - you are correct, we should not assume a level. Fixed in  dfb81aa7ff6d85566b7d95f1fa39151352cc9c67

**Comment by cavac** [comment]
The test still looks incorrect after the patch. It now tests two isolation levels. But it could be ANY of the valid levels in postgresql. If there is a single database application running, the level often gets configured by the sysadmin or developer to whatever the application design requires.

Those are:

- read uncommited
- read committed
- repeatable read
- serializable
